config.py

```diff
@@ -0,0 +1,70 @@
+from typing import Dict,Union
+import pieces_os_client as pos_client
+import urllib.request
+import json
+import sublime
+
+
+def get_models_ids() -> Dict[str, Dict[str, Union[str, int]]]:
+    # api_instance = pos_client.ModelsApi(api_client)
+
+    # api_response = api_instance.models_snapshot()
+    # models = {model.name: {"uuid":model.id,"word_limit":model.max_tokens.input} for model in api_response.iterable if model.cloud or model.downloading} # getting the models that are available in the cloud or is downloaded
+    
+
+    
+    # call the api until the sdks updated
+    response = urllib.request.urlopen('http://localhost:1000/models').read()
+    response = json.loads(response)["iterable"]
+    models = {model["name"]:model["id"] for model in response if model["cloud"] or model.get("downloaded",False)}
+    return models
+
+
+
+
+def on_settings_change():
+    global host,api_client,ASSETS_IDENTIFIERS_WS_URL
```

Comment:
let's make this a class and move these to class properties / static class properties so we avoid the use of globals